CMakeLists.txt

```diff
@@ -107,9 +109,9 @@ target_compile_options(${PROJECT_NAME}
     "-Wextra-semi"
 )
 
-add_custom_command(TARGET ${PROJECT_NAME} POST_BUILD
-  COMMAND ${CMAKE_COMMAND} -E copy_if_different $<TARGET_RUNTIME_DLLS:${PROJECT_NAME}> $<TARGET_FILE_DIR:${PROJECT_NAME}>
-  COMMAND_EXPAND_LISTS
-)
+#add_custom_command(TARGET ${PROJECT_NAME} POST_BUILD
+#  COMMAND ${CMAKE_COMMAND} -E copy_if_different $<TARGET_RUNTIME_DLLS:${PROJECT_NAME}> $<TARGET_FILE_DIR:${PROJECT_NAME}>
+#  COMMAND_EXPAND_LISTS
+#)
```

Comment:
Yes totaly. It kill your compilation sheme. I suggest to create another CMakeFiles_Linux.txt to avoid this.
With SDL2 libraries, i can write elabored CMakeFiles to avoid split this file in two.